moncli/types.py

```diff
@@ -300,6 +302,72 @@ def _export(self, value):
         return str(value)
 
 
+class PeopleType(MondayType):
+
+    native_type = list
+    native_default = []
+    allow_casts = (int, str, dict)
+    null_value = {}
+    max_allowed = 0
+
+    def __init__(self, id: str = None, title: str = None, max_allowed: int = 0, *args, **kwargs):
+        self.max_allowed = max_allowed
+        if max_allowed > 4 or max_allowed < 0:
+            self.max_allowed = 0
+        if max_allowed == 1:
+            self.native_type = en.cv.PersonOrTeam
+            self.native_default = None
+        super().__init__(id, title, *args, **kwargs)
+
+    def _process(self, value):
```

Comment:
This works.  Here is a simplified version that utilizes the assumptions of the data coming in.

```python
    def _process(self, value):
        if isinstance(value,en.cv.PersonOrTeam):
            return value
        return_list = []
        for data in value:
            if isinstance(data,en.cv.PersonOrTeam):
                return_list.append(data)
                continue
            try:
                if isinstance(data, dict):
                    id = int(data['id'])
                    kind = enums.PeopleKind[data['kind']]
                    return_list.append(en.cv.PersonOrTeam(id=id,kind=kind))               
                elif isinstance(data, (str,int)):
                    return_list.append(en.cv.Person(int(data)))
                else:
                    raise ValueError('')
            except (ValueError, KeyError):
                raise ConversionError('Cannot convert value "{}" to Person or Team.'.format(data))
        return return_list
```